Sources/TuistKit/Services/DependenciesService.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+import TSCBasic
+import TuistCore
+import TuistDependencies
+import TuistLoader
+import TuistSupport
+
+final class DependenciesService {
+    private let dependenciesController: DependenciesControlling
+    private let dependenciesModelLoader: DependenciesModelLoading
+
+    init(dependenciesController: DependenciesControlling = DependenciesController(),
+         dependenciesModelLoader: DependenciesModelLoading = DependenciesModelLoader())
+    {
+        self.dependenciesController = dependenciesController
+        self.dependenciesModelLoader = dependenciesModelLoader
+    }
+
+    func run(path: String?, method: InstallDependenciesMethod) throws {
+        switch method {
+        case .fetch: logger.info("Start fetching dependencies.", metadata: .section)
```

Comment:
When we are printing logs, we are talking to the users and therefore we should refer to the second person and not the infinitive. I suggest changing the logs here to something along the lines of:

```
We are starting to fetch/update the dependencies.
```